docs/userGuide/syntax/annotations.md

```diff
@@ -0,0 +1,145 @@
+| color     | `String` | `'green'`   | The color of the point.<br>Supports any color in the CSS color format. E.g. `red`, `#ffffff`, `rgb(66, 135, 245)`, etc...                |
+| opacity   | `String` | `'0.3'`     | The opacity of the point.<br>Supports range of values from `0` to `1`.                                                                   |
+| fontSize  | `String` | `'14'`      | The font size of the label.<br>Supports any pixel size smaller than size of the point.                                                   |
+| textColor | `String` | `'black'`   | The color of the label.<br>Supports any color in the CSS color format. E.g. `red`, `#ffffff`, `rgb(66, 135, 245)`, etc..                 |
+| legend    | `String` | `'popover'` | The presence of a legend to be displayed.<br>Supports: `popover`, `bottom`, `both`.                                                      |
+
+****`<annotate>` Options****
+
+This is effectively the same as the options used for the [picture](#pictures) component.
+
+| Name   | Type      | Default | Description                                                                                                                                                                                                           |
+| ------ | --------- | ------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| alt    | `string`  |         | **This must be specified.**<br>The alternative text of the image.                                                                                                                                                     |
+| src    | `string`  |         | **This must be specified.**<br>The URL of the image.<br>The URL can be specified as absolute or relative references. More info in: _[Intra-Site Links]({{baseUrl}}/userGuide/formattingContents.html#intraSiteLinks)_ |
+| height | `string`  |`''`| The height of the image in pixels.                                                                                                                                                                                    |
+| width  | `string`  |`''`| The width of the image in pixels.<br>If both width and height are specified, width takes priority over height. It is to maintain the image's aspect ratio.                                                            |
+| eager  | `boolean` | false   | The `<pic>` component lazy loads its images by default.<br>If you want to eagerly load the images, simply specify this attribute.                                                                                     |
+
+</div>
+
+<div id="short" class="d-none">
+
+```
+<annotate src="../../images/annotateSampleImage.png" width="500" alt="sampleImage">
```

Comment:
```suggestion
<annotate src="../../images/annotateSampleImage.png" width="500" alt="Sample Image">
```
Since `alt` is usually the text displayed when the image fails to render :)